server_environment/models/server_env_tech_name_mixin.py

```diff
@@ -16,56 +17,63 @@ class ServerEnvTechNameMixin(models.AbstractModel):
     This mixin helps solve the problem by providing a tech name field
     and a cleanup machinery as well as a unique constrain.
 
-    To use this mixin add it to the _inherit attr of your module like:
-
-        _inherit = [
-            "my.model",
-            "server.env.techname.mixin",
-            "server.env.mixin",
-        ]
-
+    Use it in place of "server.env.mixin".
     """
 
     _name = "server.env.techname.mixin"
+    _inherit = "server.env.mixin"
     _description = "Server environment technical name"
     _sql_constraints = [
         ("tech_name_uniq", "unique(tech_name)", "`tech_name` must be unique!",)
     ]
-    # TODO: could leverage the new option for computable / writable fields
-    # and get rid of some onchange / read / write code.
+
     tech_name = fields.Char(
+        string="Technical Name",
```

Comment:
this does not add any extra info. I would then rename this to "Technical env name" or "Environment technical name"